First version of the doc (#274)
ServiceNow/azimuth: docs/docs/user-guide/index.md

    [**Exploration Space**](exploration-space/index.md) to explore and interact with the utterances
    and the predictions.

## Dataset Warnings

> The dataset warnings section highlights issues related to **class size**, **class imbalance** and **dataset shift**, i.e. gaps between the class distributions of the training and the evaluation sets.

Reviewer: I think we might want to define dataset shift more broadly, so that it includes things like differences in token count distributions. Does this work?

```suggestion
The dataset warnings section highlights issues related to **class size**, **class imbalance** and **dataset shift**, i.e. differences between the data distributions of the training and the evaluation sets.
```

![Screenshot](../_static/images/dashboard/dataset-class-distribution-analysis.png) #TODO

* **Missing samples**: Verify if each intent has sufficient samples in both sets.

> * **Class imbalance**: Flag if some classes are suffering from imbalance in either sets.

Reviewer: ```suggestion
* **Class imbalance**: Flag when some classes suffer from imbalance in either split.
```
Your call set vs. split, though.

* **Representation mismatch**: Assess that the representation of each intent is similar in both
  sets.
* **Length mismatch**: Verify that the utterances' length are similar for each intent in both sets.

Select `View Details` to get to [:material-link: Dataset Warnings](dataset-warnings.md).

## Prediction Quality by Data Subpopulation

This section summarizes the quality of the predictions in terms of the [**prediction

> outcomes**](../key-concepts/outcomes.md) and **metrics** available in Azimuth, for different data subpopulations. Change the value in the dropdown :material-arrow-down-drop-circle-outline: to see the metrics broken down per label, predicted class, or smart tag families. Use the toggle to alternate between the performance on the training set or on the evaluation set.

Reviewer: Do we use "set" throughout and I just keep forgetting...? Sorry if so!
